### PR TITLE
chore(admob, auth, iam, predictions): migrate to Java 8

### PR DESCRIPTION
**Pull request by rosariopfernandes**
I managed to fix the error in #164 by adding the lines below to the modules using the androidx.browser package:
```groovy
compileOptions {
        sourceCompatibility '1.8'
        targetCompatibility '1.8'
}
```

### CODE REVIEW

File: inappmessaging/app/build.gradle [hunk 1]
```diff
@@ -20,13 +20,18 @@ android {
...
+    }
 }
 
 dependencies {
     implementation 'androidx.multidex:multidex:2.0.1'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.appcompat:appcompat:1.1.0'
-    implementation 'androidx.browser:browser:1.0.0'
+    implementation 'androidx.browser:browser:1.2.0'
+    implementation 'com.google.guava:guava:28.2-android'
```
Comment (rosariopfernandes) [inline inappmessaging/app/build.gradle]: Explicitly added this dependency, as suggested [here](https://stackoverflow.com/a/56639924/5861618), to fix the error `java.lang.RuntimeException: Duplicate class com.google.common.util.concurrent.ListenableFuture found in modules guava-26.0-android.jar (com.google.guava:guava:26.0-android) and listenablefuture-1.0.jar (com.google.guava:listenablefuture:1.0)`

**Comment by samtstern** [comment]
@rosariopfernandes thanks for this!  Looks like they added the Java 8 dependency in `1.2.0-alpha08`:
https://developer.android.com/jetpack/androidx/releases/browser#1.2.0

I don't really want to update these samples to use Java 8 since I think that Java 8 adoption is pretty low (people seem to either use Java 7 or Kotlin).  I will probably just tell the bot to leave us at `browser` version 1.0.0.

Also I am going to bother the AndroidX team and ask then why they didn't make this version `2.0.0`

**Comment by rosariopfernandes** [comment]
@samtstern I see, makes sense to keep the samples in Java 7.

**Comment by samtstern** [comment]
@rosariopfernandes going to close this now, I updated the dependency bot to stop trying to bump `browser` past `1.0.0` as a short-term fix.  Thank you for helping me understand this was a Java 8 issue!

**Pull request closed without merging**